tests/unit_tests/test_image_series.py

```diff
@@ -125,3 +127,26 @@ def test_check_image_series_external_file_relative(self):
 def test_check_image_series_external_file_valid_pass_non_external():
     image_series = ImageSeries(name="TestImageSeries", rate=1.0, data=np.zeros(shape=(3, 3, 3, 3)), unit="TestUnit")
     assert check_image_series_external_file_valid(image_series=image_series) is None
+
+
+def test_check_large_image_series_stored_internally():
+
+    gb_size = 1.0
+    image_size = 10
+    total_elements = int(gb_size * 1e9 / np.dtype("float").itemsize) // (image_size * image_size)
```

Comment:
Since you're already making use of the `gb_lower_bound` argument, I'd just go ahead and make this testing example somewhat smaller (does 10 MB sound reasonable?) so it doesn't slow down the tests too much.

Reply:
Sure.